annotator/openpose/__init__.py

```diff
@@ -354,3 +356,36 @@ def __call__(
         if json_pose_callback:
             json_pose_callback(encode_poses_as_json(poses, H, W))
         return draw_poses(poses, H, W, draw_body=include_body, draw_hand=include_hand, draw_face=include_face)
+        
+class AnimalposeDetector:
```

Comment:
This class is not reference anywhere. How does it interact with rest of the code?